ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -17,9 +25,9 @@
 namespace ReactNative.Views.View
 {
     /// <summary>
-    /// View manager for React view instances.
+    /// RCTView implementation shared between WPF and UWP.
     /// </summary>
-    public class ReactViewManager : ViewParentManager<BorderedCanvas>
+    public class BaseViewManager : ViewParentManager<BorderedCanvas>
```

Comment:
>BaseViewManager [](start = 17, length = 15)

BaseViewManager is a misleading name. Maybe just ReactViewManagerBase?

Reply:
Could also consider using a partial class here.

---
In reply to: [159500922](https://github.com/Microsoft/react-native-windows/pull/1549#discussion_r159500922) [](ancestors = 159500922)

Reply:
Fixed